includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -90,15 +98,6 @@ public function __construct() {
 			$this->secret_key      = ! empty( $main_settings['test_secret_key'] ) ? $main_settings['test_secret_key'] : '';
 		}
 
-		$this->payment_methods  = [];
-		$payment_method_classes = [
-			WC_Stripe_UPE_Payment_Method_CC::class,
-		];
-		foreach ( $payment_method_classes as $payment_method_class ) {
-			$payment_method                                     = new $payment_method_class( null );
-			$this->payment_methods[ $payment_method->get_id() ] = $payment_method;
-		}
-
```

Comment:
Any specific reason why this code block was moved a few lines above?

Reply:
Just as a precaution, to make sure `payment_methods` are loaded before calling any auxiliary methods (just in case they need them)

```
...

// Load the form fields.
$this->init_form_fields();

// Load the settings.
$this->init_settings();
```